network_apis/utils/utils.py

```diff
@@ -176,26 +177,27 @@
     return secrets.token_urlsafe(64)
 
 
-def handle_bool(value):
-    """Determine which boolean value parameter should be."""
-    LOG.debug('Determine boolean value of parameter.')
-    # Allowed values
-    truthy = [True, 'true', 't', 'yes', 'y', '1', 1]
-    falsy = [False, 'false', 'f', 'no', 'n', '0', 0]
-    if isinstance(value, bool):
-        # value is already boolean
-        return value
-    elif isinstance(value, int):
-        # value is numerical, e.g. 1 or 0
-        return True if value else False
-    else:
-        # value is a string
-        if value.lower() in truthy:
-            return True
-        elif value.lower() in falsy:
-            return False
-        else:
-            raise ValueError(f'Value should be one of {truthy+falsy}')
+# Superceded by from 'distutils.util import strtobool'

```

Comment:
we can remove this code if it is superseded.

Reply:
Done